doc/chips/11.rst

```diff
@@ -0,0 +1,183 @@
+Alternative Initializer Features
+================================
+
+Status:
+  Rejected
+
+Authors:
+  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov (for Syntax 2);
+  Vassily Litvinov (for Child Fields and Initialization Completion)
+
+Abstract
+--------
+
+This document contains the features we discussed as part of the
+initializer story and chose not to incorporate.
+
+Rationale
+---------
+
+During our development of the initializer story for Chapel
+we considered a variety of features. Some of them were accepted
+and some of them were rejected. This document records some of
+the discussed-and-rejected features for posterity.
+
+Description
+-----------
+
+Alternative syntax for distinguishing Phase 1 and Phase 2 in an initializer
+***************************************************************************
+
+We considered providing dedicated syntax for distinguishing Phase 1
+and Phase 2 in an initializer. This syntax, codenamed Syntax 2,
+designates different block statements for Phase 1 and Phase 2,
+separated by the proposed "finalize" keyword.
+
+.. code-block:: chapel
+
+   proc init() {
+     ... Phase 1 code ...
+     super.init(...);
+   } finalize {
+     ... Phase 2 code ...
+   }
+
+Certain pieces of this syntax could be omitted as follows:
+
+* The super.init() call, if it takes no arguments.
+
+* The Phase 1 block statement, if contains no statements.
+  In this case, the "finalize" keyword can also be omitted.
+  Default field initializations followed by super.init() call
+  with no arguments occur in this case.
```

Comment:
What you've described is not in keeping with my description of Syntax 2 in CHIP 10 because of my above statement, which the initializers team did not object to.  Now, that may have been an oversight on the part of the reviewers - however, if it was, it should be consistent in both places.